### PR TITLE
Activity Event API

### PR DESCRIPTION
**Pull request by mfranciszkiewicz**

### CODE REVIEW

File: specs/activity-api.yaml [hunk 1]
```diff
@@ -515,16 +439,169 @@ components:
...
+
+    TransferCommand:
+      allOf:
+        - $ref: '#/components/schemas/ExeScriptCommand'
+        - type: object
+          properties:
+            transfer:
+              $ref: '#/components/schemas/TransferCommandBody'
+
+    TransferCommandBody:
```
Comment (tworec) [inline specs/activity-api.yaml:525]:
```suggestion
    TransferCommandBody:
      required:
        - from
        - to

```

File: specs/activity-api.yaml [hunk 2]
```diff
@@ -515,16 +439,169 @@ components:
...
+
+    RunCommand:
+      allOf:
+        - $ref: '#/components/schemas/ExeScriptCommand'
+        - type: object
+          properties:
+            run:
+              $ref: '#/components/schemas/RunCommandBody'
+
+    RunCommandBody:
```
Comment (tworec) [inline specs/activity-api.yaml:562]:
```suggestion
    RunCommandBody:
      required:
        - entry_point
```

File: specs/activity-api.yaml [hunk 3]
```diff
@@ -539,19 +616,92 @@ components:
         isBatchFinished:
           type: boolean
 
-    ExeScriptCommandState:
-      required:
-        - command
+    # ExeScript streaming result structures
+
+    RuntimeEvent:
```
Comment (tworec) [inline specs/activity-api.yaml:621]: anything required here?

**Comment by mfranciszkiewicz** [comment]
Already merged into `event-api/master`

File: specs/activity-api.yaml [hunk 4]
```diff
@@ -202,48 +173,35 @@ paths:
             type: string
         - name: commandIndex
           description: >
-            Wait until command with the specified index finishes. Must be accompanied by a valid "timeout" query parameter.
+            Wait until command with the specified index finishes. Must be accompanied by a valid "pollTimeout" query parameter.
```
Comment (tworec) [inline specs/activity-api.yaml:176]: @mfranciszkiewicz you are removing `timeout` in this PR, but the comment is unchanged and still requires to specify `timeout`

**Pull request closed without merging**